README.md

```diff
@@ -1,67 +1,66 @@
-﻿# Project Reunion
+﻿# Project Reunion - Calling all Windows desktop apps!
 
-[![Feature Proposals](https://img.shields.io/github/issues/microsoft/projectreunion/feature%20proposal)](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature+proposal%22)
-[![Bugs](https://img.shields.io/github/issues/microsoft/projectreunion/bug)](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+| Delight users with modern UI | Access new platform features | Down-level support | 
+|:--:|:--:|:--:|
+| <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-get-started-get-started.svg" width=250 alt="Modern navigation"/></br>Powerful WinUI 3 UI | <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-get-started-scenarios.svg" width=250 alt="Platform logo"/></br>Localization + power status + more</br> | <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-host-integration-install-configure.svg" width=250 alt="Down-level logo"/></br>Back to Windows 10 1809 |

```

Comment:
Nit on the html - should probably be `<br/>` - that's the "element with no child elements" format; `</br>` closes a previously-opened `<br>` element.

Reply:
Thanks Rafael, `<br>` it is! Done